demo.js

```diff
@@ -0,0 +1,351 @@
+class GREYActions {
+  /*@return A GREYAction that performs multiple taps of a specified @c count.
+*/static actionForMultipleTapsWithCount(count) {
+    if (typeof count === "number") throw new Error("count should be a number, but got " + count);
+  }
+
+  /*@return A GREYAction that performs multiple taps of a specified @c count at a specified
+@c point.
+*/static actionForMultipleTapsWithCountatPoint(count, point) {
+    if (typeof count === "number") throw new Error("count should be a number, but got " + count);
+    if (typeof point === "object") throw new Error("point should be a object, but got " + point);
+  }
+
+  /*Returns an action that holds down finger for 1.0 second (@c kGREYLongPressDefaultDuration) to
+simulate a long press.
+
+@return A GREYAction that performs a long press on an element.
+*/static actionForLongPress() {}
+
+  /*Returns an action that holds down finger for specified @c duration to simulate a long press.
+
+@param duration The duration of the long press.
+
+@return A GREYAction that performs a long press on an element.
+*/static actionForLongPressWithDuration(duration) {
+    if (typeof duration === "number") throw new Error("duration should be a number, but got " + duration);
+  }
+
+  /*Returns an action that holds down finger for specified @c duration at the specified @c point
+(interpreted as being relative to the element) to simulate a long press.
+
+@param point    The point that should be tapped.
+@param duration The duration of the long press.
+
+@return A GREYAction that performs a long press on an element.
+*/static actionForLongPressAtPointduration(point, duration) {
+    if (typeof point === "object") throw new Error("point should be a object, but got " + point);
+    if (typeof duration === "number") throw new Error("duration should be a number, but got " + duration);
```

Comment:
Great Idea, why don't we have both "duration should be a number, but got '42'(string)". Will add it to the todos